app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -294,6 +294,9 @@ class BrowserTabViewModel @Inject constructor(
 
     private val replyProxyMap = mutableMapOf<String, JavaScriptReplyProxy>()
 
+    // Map<String, Map<String, JavaScriptReplyProxy>>() = Map<Origin, Map<location.href, JavaScriptReplyProxy>>()
```

Comment:
delete this?

Reply:
it's meant as documentation to know what the keys are. I can delete it if you think is not necessary

Reply:
Ha, right! I scanned it and thought it was just dead code. No strong feelings either way.